src/main/java/frc/robot/commands/Autos/PreloadOnly.java

```diff
@@ -37,22 +39,39 @@ public class PreloadOnly extends SequentialCommandGroup {
   Intake subIntake;
   Shooter subShooter;
   Transfer subTransfer;
+  int position = 0;
+
+  // BLUE
+  Pose2d S1B = new Pose2d(0.602, 6.747, Rotation2d.fromDegrees(60));
+  Pose2d S2B = new Pose2d(1.360, 5.563, Rotation2d.fromDegrees(0));
+  Pose2d S3B = new Pose2d(0.602, 4.348, Rotation2d.fromDegrees(-60));
+  Pose2d[] startingPositionsBlue = { S1B, S2B, S3B };
+
+  // RED
+  Pose2d S1R = new Pose2d(constField.FIELD_LENGTH.in(Units.Meters) - 0.602, 6.747, Rotation2d.fromDegrees(120));
+  Pose2d S2R = new Pose2d(constField.FIELD_LENGTH.in(Units.Meters) - 1.360, 5.563, Rotation2d.fromDegrees(180));
+  Pose2d S3R = new Pose2d(constField.FIELD_LENGTH.in(Units.Meters) - 0.602, 4.348, Rotation2d.fromDegrees(-120));
+  Pose2d[] startingPositionsRed = { S1R, S2R, S3R };
```

Comment:
I would either rename variable to SubPlacementCenterR or add a comment saying S = sub placement, 1 = center, etc.

Just to be more clear and easier to read in the future

We should avoid these short variable names in favor of readability